src/components/gateway/useGatewayActions.tsx

```
@@ -0,0 +1,92 @@
+      {
+        id: 'edit-annotations-gateway',
+        label: t('Edit annotations'),
+        cta: launchAnnotationsModal,
+        accessReview: updateAccess,
+      },
+      {
+        id: 'kuadrant-gateway-edit-form',
+        label: t('Edit'),
+        description: t('Edit via form'),
+        cta: () =>
+          navigate(`/k8s/ns/${namespace}/${api}~Gateway/${name}/edit`),
```

ESLint

[error] Delete `⏎·········`

(prettier/prettier)